streams/src/commonTest/kotlin/com/mirego/trikot/streams/reactive/processors/TakeWhileProcessorTests.kt

```diff
@@ -0,0 +1,191 @@
+package com.mirego.trikot.streams.reactive.processors
+
+import com.mirego.trikot.streams.cancellable.CancellableManager
+import com.mirego.trikot.streams.reactive.Publishers
+import com.mirego.trikot.streams.reactive.StreamsProcessorException
+import com.mirego.trikot.streams.reactive.subscribe
+import com.mirego.trikot.streams.reactive.takeWhile
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertFailsWith
+import kotlin.test.assertTrue
+
+class TakeWhileProcessorTests {
```

Comment:
I would also add a unit test using a completable publisher as a source publisher (such as a Publishers.just(x)) to test the completion behaviour of the operator 👍🏻